README.md

```diff
@@ -1,8 +1,49 @@
-# Fractal integration for Drupal / Twig
-
-This module enables you to build a living style guide based on Fractal within
-Drupal.
-
+# Fractal integration for Twig projects
+
+This module enables you to build a living style guide based on Fractal components using the Twig templating language.
+It can either be used as a Drupal module or as a standalone Twig extension.
+
+## Installation
```

Comment:
I think this should be moved below Features for marketing purposes – first state your USPs and what you're good for, then tell me how.

Reply:
Valid point. Will move 👍